honeycombio/data_source_recipient.go

```diff
@@ -13,44 +13,109 @@ func dataSourceHoneycombioRecipient() *schema.Resource {
 	return &schema.Resource{
 		ReadContext: dataSourceHoneycombioRecipientRead,
 
+		Description: `
+'honeycombio_recipient' data source provides details about a specific recipient.
+
+The ID of an existing recipient can be used when adding recipients to triggers or burn alerts.
+
+Note: If more or less than a single match is returned by the search, Terraform will fail. Ensure
```

Comment:
Phrasing is a bit awkward here. I might prefer "Terraform will fail unless exactly one recipient is returned by the search."

Reply:
Agreed! Updated.